dali/python/nvidia/dali/external_source.py

```diff
@@ -460,6 +460,11 @@ class ExternalSource:
         .. note::
             This is applicable only when copying data to and from GPU memory.
 
+    `blocking`: bool, optional
+        If True, this operator will block until the data is available
+        (e.g. by calling ``feed_input``).  If False, the operator will raise an error,
+        if the data is not available.
```

Comment:
I would add some warning that it's intended for advanced usage.
```suggestion
    `blocking`: bool, optional
        **Advanced** if True, this operator will block until the data is available
        (e.g. by calling ``feed_input``).  If False, the operator will raise an error,
        if the data is not available.
```

Reply:
Fixed here and in the input op.